src/main/java/canvas/CanvasContext.java

```diff
@@ -120,15 +120,15 @@ public void setLinkContext(CanvasLinkContext linkContext)
     {
         this.linkContext = linkContext;
     }
-    
+
     @Override
     public String toString()
     {
-        return String.format("Canvas Context:\n\t" + 
+        return String.format("Canvas Context:\n\t" +
                              "User Context:\n\t\t%s\n\t"+
                              "Org Context:\n\t\t%s\n\t"+
                              "Environment Context:\n\t\t%s\n\t"+
-                             "Link Context:\n\t\t%s\n", 
```

Comment:
Did anything change in this file other than formatting ? Remove from PR ?

Reply:
Done.